src/components/Account/Account.tsx

```diff
@@ -1,55 +1,43 @@
-import React, { Component } from "react";
+import React, { useEffect, useState } from "react";
 import { connect } from "react-redux";
 import { Selector, Button } from "../common/ui";
 import "./Account.css";
 import { addNewAcc } from "../../actions";
 import { IAccount } from "../../types";
+import { useForm } from "react-hook-form";
 
 interface IProps {
   accounts: IAccount[];
-  getSelectedAccount: (result: any) => void;
   accountBalance: number;
-  accBalance: number;
+  // eslint-disable-next-line no-unused-vars
+  getSelectedAccount: (account: IAccount) => void;
```

Comment:
`getSelectedOption` should be properly replaced in `Selector` component and this will not be required anymore.